modules/nw-cluster-network-range-edit.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-network-range.adoc
+
+:_content-type: PROCEDURE
+[id="nw-cluster-network-range-edit_{context}"]
+= Expanding the cluster network IP address range
+
+You can expand the IP address range for the cluster network.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+* The cluster uses the OVN-Kubernetes network plugin.
+
+.Procedure
+
+. To obtain the cluster network range and host prefix for your cluster, enter the following command:
++
+[source,terminal]
+----
+$ oc get network.operator.openshift.io -o jsonpath="{.items[0].spec.clusterNetwork}"
+----
++
+.Example output
+[source,text]
+----
+[{"cidr":"10.217.0.0/22","hostPrefix":23}]
+----
+
+. To expand the cluster network IP address range, enter the following command:
++
+[source,terminal]
+----
+$ oc patch Network.config.openshift.io cluster --type='merge' --patch \
+  '{
+    "spec":{
+      "clusterNetwork": [ {"cidr":"<cidr>","hostPrefix":<prefix>} ],
+      "networkType": "OVNKubernetes"
+    }
+  }'
+----
++
+--
+where:
+
+`<cidr>`: Specifies the cluster network IP address range. You can change the CIDR mask. Do not change the IP address range.
+`<prefix>`: Specifies the current host prefix for your cluster. This value must be the same as for your running cluster.
+--
++
+.Example output
+[source,text]
+----
+network.config.openshift.io/cluster patched
+----
+
+. To confirm that the configuration is active, enter the following command. It can take several minutes for the update to apply.
```

Comment:
the config will show up as active fairly quickly (seconds), but when the change occurs on the cluster a few
operators/daemonsets will have to be rolled back out. this can take a bit of time (tens of minutes... maybe 30+ minutes)